src/data/calculate.ts

```diff
@@ -394,34 +408,73 @@ export const calculateActivityRisk = (data: CalculatorData): number | null => {
       given: string,
     ): number => (given === '' ? 1 : table[given].multiplier)
 
+    const labelFor = (
+      table: { [key: string]: FormValue },
+      given: string,
+    ): string =>
+      given === '' ? '' : table[given].label_breakdown || table[given].label
+
     let effectiveDuration = data.duration
-    multiplier *= mulFor(Distance, data.distance)
+    variables.push({
+      name: labelFor(Distance, data.distance),
+      value: mulFor(Distance, data.distance),
+    })
     if (data.distance === 'intimate') {
       // Even a brief kiss probably has a non-trivial chance of transmission.
       effectiveDuration = Math.max(effectiveDuration, intimateDurationFloor)
     } else {
       if (data.distance !== 'close') {
         // Being outdoors only helps if you're not literally breathing each
         // others' exhalation.
-        multiplier *= mulFor(Setting, data.setting)
+        variables.push({
+          name: labelFor(Setting, data.setting),
+          value: mulFor(Setting, data.setting),
+        })
       }
       // Talking modifiers not allowed when kissing.
-      multiplier *= mulFor(Voice, data.voice)
+      variables.push({
+        name: labelFor(Voice, data.voice),
+        value: mulFor(Voice, data.voice),
+      })
 
       // You can't wear a mask if you're kissing!
-      multiplier *= mulFor(TheirMask, data.theirMask)
-      multiplier *= mulFor(YourMask, data.yourMask)
-    }
-
-    multiplier *= effectiveDuration / 60.0
-    if (multiplier > MAX_ACTIVITY_RISK) {
-      multiplier = MAX_ACTIVITY_RISK
+      variables.push({
+        name: labelFor(TheirMask, data.theirMask),
+        value: mulFor(TheirMask, data.theirMask),
+      })
+      variables.push({
+        name: labelFor(YourMask, data.yourMask),
+        value: mulFor(YourMask, data.yourMask),
+      })
     }
 
-    return multiplier * vaccineMultiplier
+    variables.push({ name: 'minutes', value: effectiveDuration })
+    return variables
   } catch (e) {
+    return []
+  }
+}
+
+export const calculateActivityRisk = (data: CalculatorData): number | null => {
+  const activityRiskVariables = getActivityRiskVariables(data)
+  if (activityRiskVariables.length === 0) {
     return null
   }
+  let activityRisk = activityRiskVariables
+    .map((variable) => variable.value)
+    .reduce((accumulator, currentValue) => (accumulator *= currentValue))
+
+  if (!['partner', 'repeated'].includes(data.interaction)) {
+    activityRisk = activityRisk / 60.0
```

Comment:
Why `/ 60.0`? Please comment or use a constant.

Reply:
> This is a huge improvement, thanks so much.
> 
> I left some suggestions for the strings - in general we want to be referencing the text that they saw / selected rather than the internal enum values (for instance the `silent` key is exposed as "not talking" so we should use that language in the explanation). It would also be nice to explain a bit about why the choice is good in the explanation card (for instance, "for the airplanes air filtration" rather than "for plane").

Yeah but that makes them really long which makes the breakdown much harder to read